src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsCaseData.java

```diff
@@ -59,6 +59,7 @@
     private List<Bundle> caseBundles;
     private List<SscsDocument> sscsDocument;
     private List<SscsDocument> draftSscsDocument;
+    private DocumentLink withdrawalDocument;
```

Comment:
We would like to group elements in SscsCaseData and move it to another class, to avoid reaching max number of elements